jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/common/Overlay.java

```diff
@@ -41,12 +41,12 @@ public boolean isAbsolute() {
 
     @JDIAction("Get '{name}' z-index value")
     public int zIndex() {
-        // TODO THe value must be parsed from PART of style attrubute like "z-index: 5;"
-        // as a lot of numeric values can be used in style
-        String zIndex = core().getAttribute("style").replaceAll("^\\D*?(\\d+).*$", "$1");
-        if (!zIndex.isEmpty()) {
+        // z-index value from style attribute
+        String zIndex = core().getAttribute("style").replaceAll(".*z-index: (\\d+).*", "$1");
```

Comment:
replaceAll is not needed, there is always only one value and even in case of several values we will get some wrong value

Reply:
replaced replaceAll with replaceFirst